locales/es.json

```diff
@@ -46,170 +46,170 @@
+   "no contracts found": "no se encontraron contratos",
    "Blockchain component is disabled in the config": "El componente Blockchain está deshabilitado en la configuración",
    "Couldn't connect to an Ethereum node are you sure it's on?": "No se pudo conectar a un nodo Ethereum, ¿está seguro de que está activado?",
-   "make sure you have an Ethereum node or simulator running. e.g '%s'": "asegurese de tener un nodo o simulador Ethereum en funcionamiento. e.g '%s'",
+   "make sure you have an Ethereum node or simulator running. e.g '%s'": "asegúrese de tener un nodo o simulador Ethereum en ejecución. e.g '%s'",
    "executing": "ejecutando",
    "the transaction was rejected; this usually happens due to a throw or a require, it can also happen due to an invalid operation": "la transacción fue rechazada; esto generalmente ocurre debido a un throw o un require, también puede ocurrir debido a una operación no válida",
    "Cannot find file %s Please ensure you are running this command inside the Dapp folder": "No se puede encontrar el archivo %s Asegúrese de que está ejecutando este comando dentro de la carpeta de la DApp",
-   "no config file found at %s using default config": "archivo de configuración no encontrado en %s utilizado la configuración predefinida",
-   "HTTP contract file not found": "archivo de contracto HTTP no encontrado",
-   "contract file not found": "archivo de contracto no encontrado",
+   "no config file found at %s using default config": "archivo de configuración no encontrado en %s utilizado la configuración predeterminada",
+   "HTTP contract file not found": "archivo de contrato HTTP no encontrado",
+   "contract file not found": "archivo de contrato no encontrado",
    "file not found, creating it...": "archivo no encontrado, creándolo ...",
    "No Blockchain node found": "No se encontró ningún nodo de Blockchain",
    "Ethereum node (version unknown)": "Nodo Ethereum (versión desconocida)",
-   "this event is deprecated and will be removed in future versions %s": "este evento está en desuso y se eliminará en futuras versiones %s",
-   "Error while downloading the file": "Error descargando el archivo",
-   "Plugin {{name}} can only be loaded in the context of \"{{contextes}}\"": "El complemento {{name}} solo se puede cargar en el contexto de \"{{contextes}} \"",
+   "this event is deprecated and will be removed in future versions %s": "este evento está obsoleto y se eliminará en futuras versiones %s",
+   "Error while downloading the file": "Error al descargar el archivo",
+   "Plugin {{name}} can only be loaded in the context of \"{{contextes}}\"": "El complemento {{name}} solo puede ser cargado en el contexto de \"{{contextes}} \"",
    "error running service check": "error al ejecutar la comprobación del servicio",
    "help": "ayuda",
    "quit": "salir",
    "Type": "Tipo",
    "to see the list of available commands": "para ver la lista de comandos disponibles",
    "Asset Pipeline": "Asset Pipeline",
```

Comment:
"Conducto de Activos"

Reply:
"Tubería" might be better [https://es.wikipedia.org/wiki/Tuber%C3%ADa_(inform%C3%A1tica)](https://es.wikipedia.org/wiki/Tuber%C3%ADa_(inform%C3%A1tica)

Reply:
yes 👍